db/migrate/20190719185254_add_trigger_to_target_limit_count_by_user.rb

```diff
@@ -0,0 +1,31 @@
+class AddTriggerToTargetLimitCountByUser < ActiveRecord::Migration[5.2]
```

Comment:
As mentioned in the ruby channel, this is not really maintainable.
We could still take this into account if performance is critical.

Reply:
I think now is already done, we can keep it. Also, nice job here! 👏 👏